### PR TITLE
Detail page, all done except map component

### LINKED ISSUE #20: Plant detail

### PR DESCRIPTION
**Pull request by buitannguyen2612**
closes #20

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -39,15 +41,15 @@ export default function Header({ navItems }) {
...
                 <Box sx={{ borderBottom: 2, borderColor: 'transparent' }}>
                     <Tabs value={value} onChange={handleChange} aria-label="basic tabs example" TabIndicatorProps={{ style: { backgroundColor: "#69AD28" } }} sx={{
                         '& .Mui-selected': {
                             color: '#69AD28 !important'
                         }
                     }}>
-                        {navItems?.map((item) => (
-                            <Tab key={item} label={item} sx={{ color: "#214400", fontWeight: "700" }} />
+                        {nav?.map((item, idx) => (
+                            <Tab key={idx} label={item} sx={{ color: "#214400", fontWeight: "700" }} />
```
Comment (phunlh2001) [inline src/components/Header/Header.jsx]: It seems the links to the pages are missing here, let fix it

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -14,14 +14,16 @@ const iconStyle = {
...
-export default function Header({ navItems }) {
+export default function Header({ }) {
     const [value, setValue] = useState(0)
     const [openPf, setOpenPf] = useState(false)
 
     const handleChange = (event, newValue) => {
         setValue(newValue)
     }
 
+    const nav = ['TRANG CHỦ', 'PHÁT HIỆN HÌNH ẢNH', 'BÀI VIẾT', 'THỰC VẬT', 'VỀ CHÚNG TÔI']
```
Comment (phunlh2001) [inline src/components/Header/Header.jsx]: Are you sure the navigation bar only has the text?

File: src/components/InforDetail/InforDetail.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            </styleMui.TxtListHead>
+            <List component="nav" sx={{ width: "100%", p: "0 0 0 2.37rem" }}>
+                {/* Hứng danh mục list và dữ liệu data trả về vào một mãng object sau đó pass đi vào component khác */}
+                {[
+                    { label: 'Tên quốc tế', value: data[0].name },
+                    { label: 'Tên thường gọi', value: data[0].nameEx },
+                    { label: 'Họ của cây', value: data[0].hoCay },
+                    { label: 'Nơi sinh trưởng', value: data[0].location },
+                    { label: 'Cơ sở bán', value: data[0].shopSell },
+                ].map((item, idx) => (<InfoList key={idx} label={item.label} value={item.value} />))}
```
Comment (phunlh2001) [inline src/components/InforDetail/InforDetail.jsx]: Why don't you create a variable instead? It's very difficult to read.

File: src/components/MoreDetail/MoreDetail.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            <styleMui.TxtListHead>ĐINH LĂNG – CÂY SÂM CỦA NGƯỜI NGHÈO</styleMui.TxtListHead>
+            <List component="nav" sx={{ width: "100%" }}>
+                {/* Theo dự tính sẽ làm theo data là: render ra các bài thuốc dựa vào cây đó "Nhưng chưa có data nên dưới đây là mẫu" */}
+                    {[
+                        { label: 'Tên quốc tế', value: data[0].name },
+                        { label: 'Tên thường gọi', value: data[0].nameEx },
+                        { label: 'Họ của cây', value: data[0].hoCay },
+                        { label: 'Nơi sinh trưởng', value: data[0].location },
+                        { label: 'Cơ sở bán', value: data[0].shopSell },
+                    ].map((item, idx) => (<MoreDetailList key={idx} label={item.label} value={item.value} />))}
```
Comment (phunlh2001) [inline src/components/MoreDetail/MoreDetail.jsx]: This one is the same `InfoDetail.jsx`

Why don't you combine 2 components into one?

File: src/components/MoreDetail/MoreDetailList/MoreDetailList.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { ListItem } from '@mui/material'
+import React from 'react'
+import * as styleMui from '../MoreDetail.styled'
+
+export default function MoreDetailList({ label, value }) {
+    return (
+        <ListItem sx={{display:"flex", alignItems:"flex-start"}}>
+            <styleMui.Dot>•</styleMui.Dot>
+            <styleMui.TextList> {label}: {value}</styleMui.TextList>
+        </ListItem>
```
Comment (phunlh2001) [inline src/components/MoreDetail/MoreDetailList/MoreDetailList.jsx]: This component seems like `InforList.jsx`

You should combine 2 into 1.

File: src/components/Sologan/Sologan.styled.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { Box, Typography, styled } from '@mui/material'
 
-export const BoxContainer = styled(Box)(() => ({
+export const BoxContainer = styled(Box)(({ theme }) => ({
```
Comment (phunlh2001) [inline src/components/Sologan/Sologan.styled.js]: Why do you add a `theme` prop to here? I see you never use it.

File: src/components/Sologan/Sologan.styled.js [hunk 7]
```diff
@@ -17,7 +17,7 @@ export const BoxContainer = styled(Box)(() => ({
   padding: '1.75rem 0 2.31rem 0',
 }))
 
-export const TitleBox = styled(Typography)(() => ({
+export const TitleBox = styled(Typography)(({ theme }) => ({
```
Comment (phunlh2001) [inline src/components/Sologan/Sologan.styled.js]: In this one the same

File: src/components/Sologan/Sologan.styled.js [hunk 8]
```diff
@@ -40,11 +40,11 @@ export const TitleBox = styled(Typography)(() => ({
...
 
-export const BoxDescription = styled(Box)(() => ({
-    textAlign: "justify",
-    width: "100%",
-    fontSize: "1.375rem",
-    fontWeight: "300",
-    lineHeight: "normal",
-    color: "#214400"
-  }))
+export const BoxDescription = styled(Box)(({ theme }) => ({
```
Comment (phunlh2001) [inline src/components/Sologan/Sologan.styled.js]: Here is the same

File: src/pages/DetailPage/DetailPage.jsx [hunk 9]
```diff
@@ -1,9 +1,88 @@
...
-    </div>
+    <Box component="section" sx={{ width: "90rem", display: "flex", flexDirection: "column", alignItems: "center" }}>
+      <Header />
+      <HerosDeatail />
+      <Stack direction="row" width="90rem" height="56.0625rem" mt="3rem" sx={{ backgroundColor: "#F4FFEB" }}>
+        <SlideDetail data={data} />
+        <InforDetail data={data} />
+      </Stack>
+      <MoreDetail data={data} />
+      <MoreBlog data ={fakeData}/>
```
Comment (phunlh2001) [inline src/pages/DetailPage/DetailPage.jsx]: Let's combine `MoreDetail.jsx` and `InforDetail.jsx` into one component

**Review by phunlh2001 — CHANGES_REQUESTED** [review]
Let fix these before merge

File: src/components/Header/Header.jsx (hunk 2 above)
Comment (buitannguyen2612) [reply src/components/Header/Header.jsx]: i dont get it ?

Comment (buitannguyen2612) [reply src/components/Header/Header.jsx]: ok i got this

**Pull request merged**